src/libs/Permissions.ts

```diff
@@ -2,7 +2,7 @@ import CONST from '@src/CONST';
 import Beta from '@src/types/onyx/Beta';
 
 function canUseAllBetas(betas: Beta[]): boolean {
-    return betas?.includes(CONST.BETAS.ALL);
+    return betas.includes(CONST.BETAS.ALL);
```

Comment:
Can you quickly remove the use of `?.` throughout the file.

Reply:
are you sure we want to do that ? betas comes from onyx meaning that it can be null so removing that can cause a crash I think

Reply:
Then I think we should wrap this around an `OnyxEntry`. What do you think?

Reply:
Yeah 😄